apps/cowswap-frontend/src/modules/trade/utils/parameterizeTradeRoute.ts

```diff
@@ -29,11 +37,19 @@ export function parameterizeTradeRoute(
       params.set('buyAmount', outputCurrencyAmount)
     }
 
+    if (targetChainId) {
+      params.set('targetChainId', targetChainId)
+    }
+
     params.set('orderKind', orderKind || OrderKind.SELL)
 
     return `${path}?${params.toString()}`
   }
 
+  if (targetChainId) {
+    return `${path}?targetChainId=${targetChainId}`
```

Comment:
Wouldn't it be safer to use params.set() here too?

Reply:
Not a big difference, anyway, added URLSearchParams.